src/test/ut/ut_cortxfs_xattr_ops.c

```diff
@@ -503,6 +508,11 @@ int listxattr_test_teardown(void **state)
 
 	ut_assert_int_equal(rc, 0);
 
+	size = sizeof(ut_xattr_obj->xattr)/sizeof(ut_xattr_obj->xattr[0]);
+	for(i=0; i<size; i++)
+		free(ut_xattr_obj->xattr[i]);
```

Comment:
Pls use braces to maintain style consistency:
for (...) {
}

Reply:
fixed